libpod/runtime.go

```diff
@@ -739,11 +742,42 @@ func getLockManager(runtime *Runtime) (lock.Manager, error) {
 	return manager, nil
 }
 
+// probeConmon calls conmon --version and verifies it is a new enough version for
+// the runtime expectations podman currently has
+func probeConmon(conmonBinary string) error {
+	cmd := exec.Command(conmonBinary, "--version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+	r, err := regexp.Compile(`^conmon version (?P<Major>\d+).(?P<Minor>\d+).(?P<Patch>\d+)`)
+	if err != nil {
+		return errors.Wrapf(err, "conmon version changed format")
+	}
+	matches := r.FindStringSubmatch(out.String())
+	major, err := strconv.Atoi(matches[1])
+	if err != nil || major < 1 {
+		return define.ErrConmonOutdated
+	}
+	// conmon used to be shipped with CRI-O, and was versioned along with it.
+	// even though the conmon that came with crio-1.9 to crio-1.15 has a higher
+	// version number than conmon 1.0.0, 1.0.0 is newer, so we need this check
+	minor, err := strconv.Atoi(matches[2])
```

Comment:
Could we just go to conmon 2.0.0 and avoid this?

Reply:
My particular heart couldn't handle bare to do so

 We could have and i considered it. conmon is certainly mature enough to be considered 2.0. At this point i want this fix in 1.5 and don't want to keep everyone back with another dependency change.

Reply:
I think at this point, given I went for v1.0.0, and this work around exists, I would prefer to leave it at v1. it matches the --api-version conmon flag and an Atoi with a check is not a very expensive operation to accommodate IMO

Reply:
Yeah, but I think moving to conmon to v2.0.0 now and claiming v1.0.0 was the bits of it in CRI-O is a sellable story and it completely clears up "where was this version?" question for support issues later on down the road.

Reply:
and I'm kicking myself for not thinking fo this prior....

Reply:
I am not too worried about hitting 1.9 TBH it's pretty easy to delay with subminor patches, and then jump to 2.0 when we would go 1.9. I prefer having the built with crio-> 0.x.0->1.x.0 journey

Reply:
I too grappled with this decision, and probably should have illuminated it more. I'm kicking myself for not having this conversation earlier :D